contracts/cosmons/src/msg.rs

```diff
@@ -34,6 +34,10 @@ pub enum HandleMsg {
         token_id: String,
         msg: Option<Binary>,
     },
+    BattleMonster{
+        token_id0: String,
+        token_id1: String
```

Comment:
Maybe rename token_id0 and token_id1 in attacker_id and defender_id so parameters are more clear

Reply:
- [x] done